addons/web_editor/static/src/js/editor/odoo-editor/src/style.scss

```
@@ -266,33 +265,38 @@
 }
 @media only screen and (max-width: 767px) {
     .oe-toolbar {
-        position: relative;
+        overflow-x: auto;
         visibility: visible;
         width: 100%;
+        border-bottom: 1px solid $border-color;
         border-radius: 0;
         background-color: white;
-
+        box-shadow: none;
+        height: auto;
+        
         .btn {
             color: black;
+            padding: 3.5px 4px !important;
+        }
+        .dropdown-menu{
```

missing empty line before selector, missing empty character before opening brachet
```suggestion

        .dropdown-menu {
```